packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -113,15 +113,15 @@ const JurisdictionCard = ({ isAdded = false, isSelected = false, jurisdiction, o
                         ?.displayTextSkinColor as unknown as TDisplayTextSkinColor,
                 })}
             >
-                <Text className='leading-[1] text-system-light-primary-background text-50' weight='bold'>
+                <Text className='leading-[1] text-system-light-primary-background' size='2xs' weight='bold'>
                     {row?.titleIndicators.displayText}
                 </Text>
             </div>
         );
 
     return (
         <div
-            className={qtMerge(JurisdictionCardClass({ isAdded, isFlipped, isSelected }))}
+            className={qtMerge(JurisdictionCardClass({ isAdded, isFlipped, isSelected }), 'w-[276px]')}
```

Comment:
there are is a PR that will remove qtMerge and whatever styling you might have from quill-design. You might want to update everything just incase they merge that PR before yours.

Reply:
xmau